app/src/main/java/com/yeonae/chamelezone/view/mypage/myplace/adapter/MyPlaceRvAdapter.kt

```diff
@@ -46,7 +50,17 @@ class MyPlaceRvAdapter(private var items : ArrayList<Place>) : RecyclerView.Adap
                 tv_place_name.text = item.placeName
                 tv_place_keyword.text = item.placeKeyword
                 tv_place_address.text = item.placeAddress
+                btn_more.setOnClickListener {
+                    showPopup(btn_more)
+                }
+
             }
         }
+        private fun showPopup(v: View) {
+            val popup = PopupMenu(v.context, v)
+            val inflater: MenuInflater = popup.menuInflater
```

Comment:
```suggestion
            val inflater = popup.menuInflater
```
kotlin 타입 추론으로 생략 가능해보여요

Reply:
수정했습니다